web_ui/src/test-utils/required-providers-render.tsx

```diff
@@ -26,70 +29,83 @@ interface RequiredProvidersProps extends Partial<ApplicationServicesContextProps
     initialEntries?: string[];
     featureFlags?: CustomFeatureFlags;
     profile?: OnboardingProfile | null;
-    queryClient?: QueryClient;
 }
 
 const prefilledOrgId = '000000000000000000000001';
 
-const usePrefilledQueryClient = (featureFlags?: CustomFeatureFlags, profile?: OnboardingProfile | null) => {
-    const queryClient = new QueryClient({
-        defaultOptions: {
-            queries: {
-                gcTime: 1000,
+const PrefilledQueryClientProvider = ({
+    children,
+    featureFlags,
+    profile,
+}: {
+    children: ReactNode;
+    featureFlags?: CustomFeatureFlags;
+    profile?: OnboardingProfile | null;
+}) => {
+    const { addNotification } = useNotification();
```

Comment:
Do we need this notification behaviour in our tests? If we verify that `createGetiQueryClient` works as expected with a single unit test then I don't think we really need it here.